fixtures/features/api-elements/extensions.json

```diff
@@ -14,6 +14,26 @@
       "attributes": {},
       "content": [
         {
+          "element": "extension",
+          "meta": {
+            "links": [
+              {
+                "element": "link",
+                "meta": {},
+                "attributes": {
+                  "relation": "profile",
+                  "href": "http://swagger.io/specification/#vendorExtensions"
```

Comment:
@pksunkara absolutely. :) Something to keep in your back pocket if the presence of the extension does not convey the origin of the vendor extensions.